### PR TITLE
Buttons added

### PR DESCRIPTION
**Pull request by ayush25102001**
#32
Screenshot (14)

### CODE REVIEW

**Comment by ayush25102001** [comment]
Please review @rohan-kulkarni-25

**Comment by ayush25102001** [comment]
@pranav2012 please review

**Comment by ayush25102001** [comment]
PR for issue #32

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+package-lock.json
```
Comment (pranav2012) [inline .gitignore:2]: don't add package-lock in .gitignore

File: client/.gitignore [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+package-lock.json
```
Comment (pranav2012) [inline client/.gitignore:2]: don't add package-lock in .gitignore

**Review by pranav2012 — CHANGES_REQUESTED** [review]
you have removed mui and used bootstrap instead
use mui for adding the buttons

**Comment by ayush25102001** [comment]
> you have removed mui and used bootstrap instead
> use mui for adding the buttons

Ok sir!!

**Comment by pranav2012** [comment]
> > you have removed mui and used bootstrap instead
> > use mui for adding the buttons
> 
> Ok sir!!

any updates @ayush25102001 ?

**Comment by ayush25102001** [comment]
> > > you have removed mui and used bootstrap instead
> > > use mui for adding the buttons
> > 
> > 
> > Ok sir!!
> 
> any updates @ayush25102001 ?

I don’t know mui I am trying

**Comment by adityabisoi** [comment]
@ayush25102001 Please resolve the conflicts.

**Comment by ayush25102001** [comment]
> @ayush25102001 Please resolve the conflicts.

@adityabisoi Sir ,I have resolved the conflicts if u r fine with buttons using bootstrap instead of mui please merge

**Pull request merged**